== Future prospects

It is very likely that exposing sequence-derived data through biodiversity discovery platforms will remain in demand and grow; at the moment of writing the interest for this in the biodiversity informatics community is very high. We expect that mapping recommendations provided here will remain valid and evolve slowly, while packaging and indexing by biodiversity data platforms will develop rapidly.

We suggest that data discovery platforms such as ALA and GBIF work towards adopting data formats that support more complex relational and hierarchical data. Examples could be the https://frictionlessdata.io/[Frictionless Data Format] and the more domain specific https://biom-format.org/[Biological Observation Matrix] (BIOM) format. The latter is used by several bioinformatic tools (QIIME, Mothur, USEARCH etc.) and hence this could help publishers skip a conversion step into DwC archives. A more flexible data format than the current DwC star schema is crucial for allowing hierarchical sampling events and material samples as well as attaching sequence data to individual occurrences within a sampling event. 

There is a need to be able to easily in- or exclude sequence-derived occurrence data in biodiversity discovery platforms such as ALA and GBIF. The above suggested data formats will open opportunities for a richer classification of the types of evidence a specific occurrence record is based on. However, for the time being there is a lack of an appropriate value in the BasisOfRecord vocabulary for these data types. We suggest, as a pragmatic immediate solution, that the BasisOfRecord is extended with a value such as “DNA”, “sequence-derived”, or similar. As described above, sequence-derived data may come from well-documented sampling or individual organisms, may be backed by preserved physical material or not, and result from genetic sequencing or other DNA detection methods, such as qPCR. Biodiversity discovery platforms and TDWG need to provide means to differentiate between these data types and their origins.

We also recommend that the data discovery platforms index the actual sequences or at least a MD5 checksum of these to facilitate searches for ASVs across datasets. If ASVs are provided, MD5s should be generated by the biodiversity discovery platforms, and if ASVs are not provided, MD5s need to be mandatory.

As mentioned in <<#,Sections 2.6>> and <<#,3.1 (Category IV)>>, we encourage the data discovery platforms to continue work on adopting relevant molecular taxonomic reference databases into their taxonomic backbones.

New and emerging methods and technologies, such as Oxford Nanopore, PacBio and shotgun sequencing, will likely trigger the need for adjustments to this guide to accommodate specific new data and metadata fields.
