[glossary]
== Glossary

[glossary]
[[ala]]Atlas of Living Australia (ALA):: The ALA is a web-based platform that pulls together Australian biodiversity data from multiple sources, making it accessible and reusable to anyone (see https://www.ala.org.au/about-ala/). The open infrastructure platform developed by the ALA is also used by several other countries for their own national biodiversity data platform (see https://living-atlases.gbif.org/).
[[asv]]Amplicon Sequence Variant (ASV):: Unique DNA sequence derived from high-throughput sequencing and denoising, and assumed to represent a biologically real sequence variant. See also <<otu,Operational Taxonomic Unit (OTU)>>.
[[api]]Application Programming Interface (API):: Set of protocols and tools for interaction and data transmission between different computer applications.
[[bins]]Barcode Index Numbers (BINs):: Species-level <<otu,Operational Taxonomic Units (OTUs)>> derived from clustering of the cytochrome c oxidase I (COI) gene in animals. Each BIN is assigned a globally unique identifier, and is made available in searchable database within the <<bold,Barcode of Life Data System (BOLD)>>.
[[bold]]Barcode of Life Data System (BOLD):: Freely available sequence repository and informatics workbench for DNA barcoding data, maintained by the International Barcode of Life project (iBOL). Includes a searchable database of <<bins,Barcode Index Numbers (BINs)>>.
[[biodiversity-data-platform]]Biodiversity data platform:: General online resource to discover and access biodiversity data derived from various sources, such as natural history collections, citizen science, ecology and monitoring projects, and genetic sequences. Can be global (<<gbif,GBIF>>) or national (<<ala,ALA>>).
[[clustering]]Clustering:: In taxonomic classification, the process of grouping organisms together according to some similarity criterion. See <<otu,Operational Taxonomic Unit>>.
[[community-dna]]Community (bulk) DNA:: DNA from bulk samples (e.g. plankton samples or Malaise trap samples consisting of several individuals from many species). For the purpose of this guide, bulk sample DNA is included in the eDNA concept.
[[dwc-a]]Darwin Core Archive (DwC-A):: Compressed (ZIP) file format for exchange of biodiversity data compiled in accordance with the <<dwc-standard,Darwin Core (DwC) standard>>. Essentially a self-contained set of interconnected CSV files and an XML document describing included files and data columns, and their mutal relationships.
[[dwc-standard]]Darwin Core (DwC) standard:: Standard for sharing and publishing biodiversity data, originating from the Biodiversity Information Standards (TDWG) community. In principle, a set of terms used for describing different entities of biodiversity observations, such as sampling events, occurrences and taxa. Current Darwin Core terms are described in the https://dwc.tdwg.org/terms/[Quick Reference Guide].
[[data-vocabulary]]Data vocabulary:: Set of preferred terms or concepts
with specific, well-defined meanings and interrelationships, facilitating data exchange and reuse.
[[ddpcr]]ddPCR (droplet digital Polymerase Chain Reaction):: Droplet digital <<pcr,PCR>>. Method for measuring absolute amount of DNA (number of copies) of one marker in a sample. See also <<qpcr,qPCR>>.
[[denoising]]Denoising:: In metabarcoding, method for separation of true biological sequences (see <<asv,ASVs>>) from spurious sequence variants caused by PCR amplification and sequencing error.
[[doi]]Digital Object Identifier (DOI):: asdf.
[[barcoding]]DNA barcoding and metabarcoding (amplicon sequencing):: Use of short, standardized DNA fragments to identify individual organisms via sequencing. Metabarcoding combines barcoding with high-throughput DNA sequencing, using universal primers to amplify and sequence large groups of organisms in eDNA samples.
[[dna-marker]]DNA marker:: A DNA fragment used as a marker of some property (e.g., taxonomic affiliation). May, but does not have to, be a gene or a part of a gene.
[[dna-db]]DNA metabarcoding database:: Database containing DNA sequences (DNA barcodes) from previously recovered or studied organisms. The reference sequences were ideally generated from individuals of described, well-studied species-with the type specimen serving as the ideal-or higher taxonomic level (e.g., genus, family), but may also stem from eDNA sequencing efforts.It is wise not to trust “reference sequences” blindly.
[[dna-probe]]DNA probe:: A short, synthetic single-stranded DNA fragment with fluorescent labeling that binds to a selected region of target DNA (marker) during PCR. Increases specificity and can be used in addition to primers in <<qpcr,qPCR>> and <<ddpcr,ddPCR>> to detect and quantify a genetic marker.
[[embl]]European Bioinformatics Institute (EMBL-EBI):: Intergovernmental organization for bioinformatics research and services, part of the European Molecular Biology Laboratory (EMBL), providing eg. (raw) sequence reads and assembly data via <<ena,the Europena Nucleotide Archive (ENA)>>.
[[edna]]Environmental DNA (eDNA):: DNA from an environmental sample, e.g. soil, water, air or host organism. An often used definition is that environmental DNA is the genetic material (DNA) obtained from environmental samples without any obvious evidence of biological source material (https://doi.org/10.1016/j.biocon.2014.11.019[Thomsen and Willerslev 2015^]).
[[ena]]European Nucleotide Archive (ENA):: European repository for nucleotide sequences, covering raw sequencing data, sequence assembly information and functional annotation. Includes the <<sra,Sequence Read Archive (SRA)>>, and is maintained by the European Bioinformatics Institute (EMBL-EBI), as part of <<insdc, the International Nucleotide Sequence Database Collaboration (INSDC)>>.
[[fastq]]FASTQ:: Text-based standard for storing molecular sequences and associated quality measures deriving from <<hts,High-throughput sequencing (HTS)>>. For each sequence position, single ASCII-characters are used to represent base call (identified nuctleotide) and score, respectively.
[[gbif]]GBIF:: asdf.
[[ggbn]]Global Genome Biodiversity Network (GGBN):: International network of institutions concerned with efficient sharing and usage of genomic biodiversity samples and associated metadata, e.g. promoting the Darwin Core-compatible GGBN Data Standard.
[[gps]]Global Positioning System (GPS):: Satellite navigation system operated by the United States Space Force.
[[hts]]High-throughput sequencing (HTS):: Different technologies for massively parallel sequencing, producing millions of DNA sequence reads from library preparations of genetic material, rather than targeting single amplicons as in traditional Sanger sequencing. Also called Next Generation Sequencing (NGS).
[[ingest]]Ingestion:: Process of importing data from heterogeneous sources, such as local databases, text files or spreadsheets, to a common destination system, such as an online <<biodiversity-data-platform, biodiversity data platform>>, for storage and further analysis. Typically includes steps of extraction, transformation (cleaning) and loading (ETL).
[[index]]Indexing:: asdf.
[[insdc]]International Nucleotide Sequence Database Collaboration (INSDC):: asdf.
[[metagenomics]]Metagenomics:: PCR-free sequencing of random genomic fragments in a mixed sample.
[[mixs]]Minimum Information about any (x) Sequence (MIxS) standard:: Family of standards (checklists) for sequence metadata, developed by the Genomic Standards Consortium (GSC).
[[motu]]molecular Operational Taxonomic Unit (mOTU):: See <<otu,Operational Taxonomic Unit (OTU)>>.
[[ncbi]]National Center for Biotechnology Information (NCBI):: Division of United States National Library of Medicine (NLM) housing important bioinformatics resources, such as the GenBank database of DNA sequences, and the <<sra,Sequence Read Archive (SRA)>> of high throughput sequencing data.
[[ngs]]Next Generation Sequencing (NGS):: See <<hts,High-throughput sequencing (HTS)>>.
[[occurrence]]Occurrence:: asdf.
[[otu]]Operational Taxonomic Unit (OTU):: Cluster of organisms based on similarity in specific DNA marker sequence(s), used for taxonomic classification. Includes, for example, <<sh,Species Hypothesis>> in UNITE, and <<bins,Barcode Index Numbers>> in the Barcode of Life Data System (BOLD). <<asv,Amplicon Sequence Variants (ASVs)>> may be considered analogous to <<zotu,zero radius OTUs (zOTUs)>>.
[[pcr]]Polymerase Chain Reaction (PCR):: Technique for fast amplification and detection of specific fragments of target DNA (or RNA) sequences. Amplified regions are determined by the pair of <<primer,PCR primers>> used in the reaction.
[[pipeline]]Pipeline:: asdf.
[[primer]]Primers (PCR primers):: Short, synthetic, single-stranded DNA fragments that bind to a selected region of target DNA (marker) to initiate replication during <<pcr,PCR>>. A pair of primers is necessary for the polymerase enzyme to amplify the selected marker.
[[qpcr]]qPCR (quantitative Polymerase Chain Reaction):: Quantitative <<pcr,PCR>>. Method that measures relative DNA quantity of a marker in a sample. See also <<ddpcr,ddPCR>>.
[[sample]]Sample:: asdf.
[[seq-align]]Sequence alignment:: Bioinformatic process of comparing and arranging two or more molecular (DNA, RNA or protein) sequences to detect similarities caused by e.g. evolutionary relatedness.
[[sh]]Species Hypothesis (SH):: Species-level <<otu,Operational Taxonomic Unit (OTU)>> as defined in the UNITE database and sequence management environment, for Fungi.
[[specimen]]Specimen:: asdf.
[[sra]]Sequence Read Archive (SRA):: Public repository of high throughput (<<ngs,NGS>>) sequencing data, with instances operated by <<ncbi,the National Center for Biotechnology Information (NCBI)>>, <<embl,the European Bioinformatics Institute (EMBL-EBI)>>, and the DNA Data Bank of Japan (DDBJ). Includes both raw (non-denoised) sequencing output and <<seq-align,sequence alignments>>. One of three components of <<ena, the European Nucleotide Archive (ENA)>>, and previously known as the Short Read Archive.
[[tcs]]Target-capture sequencing:: Sequencing of DNA fragments isolated with hybridization probes.
[[zotu]]Zero radius otu (zOTU):: See <<asv,ASV>>.

<<<
